etc/discovery-client/discovery-client.yaml

```diff
@@ -4,6 +4,7 @@ internalDir: /etc/discovery-client/internal/
 reconnectInterval: 5s
 logPagePaginationEnabled: false
 maxIOQueues: 0
+kato: 20
```

Comment:
why set default to 10 and yaml to 20?
is 10 the nvme client default value? and you want to keep this for backward compatibility.

though I see yaml is part of spec file, so wont changing the rpm, also anyways update the yaml?

Reply:
It's mainly an example and intentionally I set it different than default. but I don't mind changing it.

Reply:
not that critical, just want to be clear on what user will get once he uses rpm, I just wanted clarification if we want value 10 or 20 in actual production.
possibly misunderstood this but as spec file doesnt use "norplace" for the yaml, will this overwirte user existing yaml on upgrade of the binary, in which case only value from yaml (20) will be used?